src/components/home.js

```diff
@@ -36,10 +36,41 @@ const Home = () => {
           {
             bookList.bookItems.map((book) => (
               <li className="book" key={book.id}>
-                <p className="category">{book.cat}</p>
-                <p className="tittle">{book.tittle}</p>
-                <p className="author">{book.author}</p>
-                <button type="button" className="remove" onClick={() => deleteBook(book.id)}>Remove</button>
+                <div className="book-content">
+                  <p className="category">{book.cat}</p>
+                  <p className="tittle">{book.tittle}</p>
+                  <p className="author">{book.author}</p>
+                  <div className="buttons">
+                    <button type="button" className="remove">Comments</button>
+                    <button type="button" className="remove" onClick={() => deleteBook(book.id)}>Remove</button>
+                    <button type="button" className="remove">Edit</button>
```

Comment:
- [ ] It would be better to add the border between comment remove and edit to your design this will make your design look like the original 🤓